src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -0,0 +1,37 @@
+import type { UtxosFee } from '$btc/types/btc-send';
+import { convertNumberToSatoshis } from '$btc/utils/btc-send.utils';
+import { selectUserUtxosFee } from '$lib/api/backend.api';
+import type { BtcAddress } from '$lib/types/address';
+import { mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
+import type { Identity } from '@dfinity/agent';
+import type { BitcoinNetwork } from '@dfinity/ckbtc';
+
+const DEFAULT_MIN_CONFIRMATIONS = 6;
+
+export const selectUtxosFee = async ({
+	identity,
+	sourceAddress,
+	network,
+	amount
+}: {
+	identity: Identity;
+	network: BitcoinNetwork;
+	sourceAddress: BtcAddress;
+	amount: number;
+}): Promise<UtxosFee> => {
```

Comment:
Shouldn't this return a `ResultSuccess`

Reply:
This is the call that will be triggered on the SendReview step, the response is needed to display the fee amount in the modal and to start the transaction in case user proceeds with the sending.

Reply:
Given that you are going to handle the try / catch in the UI it's fine indeed